tequilapi/endpoints/identities.go

```diff
@@ -101,6 +106,22 @@ func (endpoint *identitiesApi) Register(resp http.ResponseWriter, request *http.
 	resp.WriteHeader(http.StatusAccepted)
 }
 
+func (endpoint *identitiesApi) Unlock(resp http.ResponseWriter, request *http.Request, params httprouter.Params) {
+	id := params.ByName("id")
+	unlockReq, err := toUnlockRequest(request)
+	if err != nil {
+		utils.SendError(resp, err, http.StatusBadRequest)
+		return
+	}
+
+	err = endpoint.idm.Unlock(id, unlockReq.Passphrase)
+	if err != nil {
+		utils.SendError(resp, err, http.StatusUnprocessableEntity)
```

Comment:
It not validation error, but failed action e.g. 500

Reply:
500 seems to harsh, becaus the most likely reason for this failure is user error - user passed us incorrect passphrase. Any other suitable http statuses?

Reply:
422 unprocessable entity?

Reply:
403 Forbidden

Reply:
Done 403